content/rancher-prime/aws/contents.lr

```diff
@@ -0,0 +1,323 @@
+  generation: 480
+  name: rancher-usage-record
+  resourceVersion: "430013"
+  uid: cf47eb21-54ae-4a35-8e75-a2362a9baa4f
+reporting_time: "2023-10-04T00:44:37Z"
+```
+
+## Upgrading a Rancher Prime PAYG Cluster
+
+The marketplace PAYG offer is tied to a billing adapter AND Rancher Prime version. These are updated periodically as new version of the billing adapter or Rancher are released.
+In that case the helm chart will be updated with new tags and digests and new version of helm chart will be uploaded. In order to upgrade the deployed helm chart with new version, execute the following helm command
+
+```
+helm upgrade -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \
+oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/rancher-payg/rancher-cloud-helm/rancher-cloud \
+  --version <upgraded_chart_version> \
+  --set rancherHostname=$HOST_NAME \
+  --set rancherServerURL=https://$HOST_NAME \
+  --set rancherReplicas=$REPLICAS \
+  --set rancherBootstrapPassword=$BOOTSTRAP_PASSWORD \
+  --set global.aws.accountNumber=$AWS_ACCOUNT_ID \
+  --set global.aws.roleName=$ROLE_NAME
+```
+
+To check if upgraded helm chart installation is deployed, run following command. It should display the upgraded chart version and REVISION incremented by 1 from previous install.
+
+```
+helm ls -n cattle-rancher-csp-deployer-system
+```
+
+## Troubleshooting
+
+This section contains information to help you troubleshoot issues when install Rancher and configure Billing-adapter
+
+After successful deployment, it should list similar pod and chart output
+
+```
+kubectl get deployments --all-namespaces=true
+```
+
+```
+NAMESPACE                           NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+cattle-csp-billing-adapter-system   csp-rancher-usage-operator    1/1     1            1           8h
+cattle-csp-billing-adapter-system   rancher-csp-billing-adapter   1/1     1            1           8h
+cattle-fleet-local-system           fleet-agent                   1/1     1            1           8h
+cattle-fleet-system                 fleet-controller              1/1     1            1           8h
+cattle-fleet-system                 gitjob                        1/1     1            1           8h
+cattle-provisioning-capi-system     capi-controller-manager       1/1     1            1           8h
+cattle-system                       rancher                       3/3     3            3           8h
+cattle-system                       rancher-webhook               1/1     1            1           8h
+cert-manager                        cert-manager                  1/1     1            1           8h
+cert-manager                        cert-manager-cainjector       1/1     1            1           8h
+cert-manager                        cert-manager-webhook          1/1     1            1           8h
+default                             external-dns                  1/1     1            1           9h
```

Comment:
based on the conversation for the Azure docs, remove the external-dns item from this list